src/makeRobotAccountant.js

```diff
@@ -7,6 +7,26 @@
 
 function makeRobotAccountant() {
   // write code here
+  // random comment cuz i forgot to switch branch and need to make changes :P
+  let counter = 0;
+
+  function getSum(a) {
+    function bullshitNesting(b) {
+      if (counter > 2 && counter % 2 === 1) {
+        counter++;
+
+        return `Bzzz... Error!`;
```

Comment:
would wrap it in a variable, e.g. error, then return error

Reply:
Actually, since we are not doing operations on this variable, it does not change or appear in different places, it's not a part of a list of options (like in stateful clones) we don't need to wrap it in a variable.